lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -866,6 +866,14 @@ bool SessionStateEnforcer::insert_pdr_from_core(
   rule.mutable_activate_flow_req()->set_downlink_tunnel(teid);
   rule.mutable_deactivate_flow_req()->set_downlink_tunnel(teid);
 
+  /*Set the AMBR QCI Value */

```

Comment:
Same set of lines repeated in another function. Can we move these to a new function and call that function from two places wherever it needed.

Reply:
Agreed. Moved to common place function.